modules/dnn/test/test_layers_1d.cpp

```diff
@@ -245,4 +245,36 @@ INSTANTIATE_TEST_CASE_P(/*nothing*/, Layer_Elemwise_1d_Test, Combine(
 /*operation*/           Values("div", "prod", "max", "min", "sum")
 ));
 
+// WORKING FOR 1D
+typedef testing::TestWithParam<tuple<int>> Layer_Split_Test;
+TEST_P(Layer_Split_Test, Accuracy)
+{
+    LayerParams lp;
+    lp.type = "Split";
+    lp.name = "SplitLayer";
+    int top_count = 2; // 2 is for simplicity
+    lp.set("top_count", top_count);
+    int dims = get<0>(GetParam());
+
+    Ptr<SplitLayer> layer = SplitLayer::create(lp);
+    std::vector<int> input_shape = {dims};
+    std::vector<int> output_shape = {dims};
+
+    Mat input(dims, input_shape.data(), CV_32F);
+    cv::randn(input, 0.0, 1.0);
+    cv::Mat output_ref = cv::Mat(dims, output_shape.data(), CV_32F, input.data);
+
+    std::vector<Mat> inputs{input};
+    std::vector<Mat> outputs;
+
+    runLayer(layer, inputs, outputs);
+    for (int i = 0; i < top_count; i++)
+    {
+        ASSERT_EQ(shape(output_ref), shape(outputs[i]));
+        normAssert(output_ref, outputs[i]);
+    }
+}
+INSTANTIATE_TEST_CASE_P(/*nothing*/, Layer_Split_Test,
+/*input blob shape*/    Values(0, 1));
```

Comment:
I think it's critical here to add 1D but with multiple values here (see https://github.com/opencv/opencv/pull/25074#discussion_r1522654268). Even with `{2}` to check that Split will really split 1D vector to two parts.

Reply:
`Split` later just copies one tensor into multiple([see](https://github.com/opencv/opencv/blob/60ca3d259eff77bf6287c07bc8db66078d5c6cd3/modules/dnn/src/layers/split_layer.cpp#L94C5-L105C7)). The one you mention is a `slice`.

Reply:
Got it, thanks. Very strange layer, to be honest :) But let's keep test enabled